ci/hbase-data-ingestion/jobs/historic-data-import/production-import-only.yml

```diff
@@ -23,11 +23,22 @@ jobs:
           - .: (( inject meta.plan.terraform-output-snapshot-sender ))
             params:
               TF_WORKSPACE: "production"
+      - .: (( inject meta-hbase-data-ingestion.plan.update-dynamo-db-table-settings ))
+        task: update-dynamo-db-table-settings-write-heavy
+        params:
+          TYPE: "write_heavy"
+          AWS_ROLE_ARN: arn:aws:iam::((aws_account.production)):role/ci
       - .: (( inject meta-hbase-data-ingestion.plan.run-data-load ))
         config:
           params:
             AWS_ACC: ((aws_account.production))
             AWS_ROLE_ARN: arn:aws:iam::((aws_account.production)):role/ci
             ACTION_TAG: "@admin-historic-data-import-import-only"
             HISTORIC_IMPORTER_USE_ONE_MESSAGE_PER_PATH: "false"
-            HISTORIC_IMPORTER_SKIP_LATER_THAN_OVERRIDE: "2020-07-10T07:00:00.000"
```

Comment:
Was this meant to be deleted?

Reply:
@connorIW no, but it's right to be deleted as it shouldn't be hardcoded in master